app/src/main/java/com/gh4a/activities/IssueActivity.java

```diff
@@ -61,16 +66,36 @@
 import java.util.Locale;
 
 public class IssueActivity extends BaseActivity implements View.OnClickListener {
+    private static final String EXTRA_OWNER = "owner";
+    private static final String EXTRA_REPO = "repo";
+    private static final String EXTRA_NUMBER = "number";
+    private static final String EXTRA_INITIAL_COMMENT = "initial_comment";
+
     public static Intent makeIntent(Context context, String login, String repoName, int number) {
         return makeIntent(context, login, repoName, number, null);
     }
+
     public static Intent makeIntent(Context context, String login, String repoName,
             int number, IntentUtils.InitialCommentMarker initialComment) {
         return new Intent(context, IssueActivity.class)
-                .putExtra("owner", login)
-                .putExtra("repo", repoName)
-                .putExtra("number", number)
-                .putExtra("initial_comment", initialComment);
+                .putExtra(EXTRA_OWNER, login)
+                .putExtra(EXTRA_REPO, repoName)
+                .putExtra(EXTRA_NUMBER, number)
+                .putExtra(EXTRA_INITIAL_COMMENT, initialComment);
+    }
+
+    public static void startTask(Context context, String login, String repoName, int number,
+            IntentUtils.InitialCommentMarker initialComment) {
+        Intent intent = makeIntent(context, login, repoName, number, initialComment);
+        TaskUtils.startMatchingOrNewTask(context, intent, new TaskUtils.IntentMatcher() {
+            @Override
+            public boolean matches(@NonNull Bundle a, @NonNull Bundle b) {
+                b.setClassLoader(getClass().getClassLoader());
```

Comment:
This was needed to solve `NoClassDefFoundError` crashes when trying to read extras from task intent. It has something to do with `IntentUtils.InitialCommentMarker` but I found no other solutions.

You can replicate that crash by:
1. Comment out this line
2. Go to issue from news feed
3. Use recents to go back to news feed
4. Go to the same issue again